pkg/test/cel/httproute_test.go

```diff
@@ -1251,6 +1251,29 @@ func TestHTTPRouteRule(t *testing.T) {
 				return rules
 			}(),
 		},
+		{
+			name:       "invalid because multiple names are repeated",
+			wantErrors: []string{"Rule name must be unique within the route"},
+			rules: []gatewayv1.HTTPRouteRule{
```

Comment:
Mind adding another test case where the duplicate values are spread out in the list and not consecutive?